SnippingWindow.xaml

```diff
@@ -1,7 +1,8 @@
 ﻿<Window x:Class="Paragon.Plugins.ScreenCapture.SnippingWindow"
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-        Title="Snipping Tool" 
+        xmlns:p="clr-namespace:ScreenSnippet.Properties"
```

Comment:
more descriptive names for namespace would be good.

Reply:
@lneir I tried changing the namespace but it cannot be changed as the resources files are located under `ScreenSnippet.Properties.Resources`